packages/shared_preferences/shared_preferences/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="io.flutter.plugins.sharedpreferencesexample">
+    package="io.flutter.plugins.sharedpreferencesexample"> 
```

Comment:
This change (adding trailing whitespace) should be reverted.

Reply:
done